api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java

```diff
@@ -497,7 +486,7 @@ public CohortDefinition getD2() {
             "endDate=${endDate},location=${location}"));
 
     cd.setCompositionString(
-        "allPatientsTxCurr AND NOT (activeAndStablePatients OR pregnantOrBreastfeedingOrTBTreatment)");
+        "NOT (activeAndStablePatients OR pregnantOrBreastfeedingOrTBTreatment)");
```

Comment:
@ynurmahomed Think you forgot to remove this NOT, or better, does this works?

Reply:
It works.